static/app/components/teamRoleUtils.tsx

```diff
@@ -0,0 +1,13 @@
+import {Fragment} from 'react';
+
+import HookOrDefault from 'sentry/components/hookOrDefault';
+import {t} from 'sentry/locale';
+
+const LabelHook = HookOrDefault({
+  hookName: 'sidebar:item-label',
+  defaultComponent: ({children}) => <Fragment>{children}</Fragment>,
+});
+
+export function TeamRoleColumnLabel() {
+  return <LabelHook id="team-roles-upsell">{t('Team Roles')}</LabelHook>;
```

Comment:
what is this ID for/where is it used?

Reply:
It's from getsentry: https://github.com/getsentry/getsentry/pull/11125/files